.github/workflows/TEST.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       - uses: actions/setup-java@v3
         with:
           distribution: 'temurin'
-          java-version: '11'
+          java-version: '17'
```

Comment:
What was the reason for changing JVM version?

Reply:
Spring Boot 3 will only work with Java version 17+

Reply:
Got it, thanks!